src/utils/camelize.js

```diff
@@ -6,11 +6,11 @@ const regExp = /[-\s]+(.)?/g
  * @param {String} str
  * @return {String}
  */
+
 export default function camelize(str) {
   return str.replace(regExp, toUpper)
 }
 
 function toUpper(match, c) {
   return c ? c.toUpperCase() : ''
 }
-
```

Comment:
we need last line always nl, there is an issue with this  https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Didn't linter warn about it? I was sure it is checking it.

Reply:
Hm, I didn't removed it. In addition, Atom add last line it automatically. I will check

Reply:
you can make it always ensure last line, actually eslint should do that as well with --fix option